README.md

```diff
@@ -214,64 +224,323 @@ Thus, this allows you to have many vendored tarballs by using the `--tag` parame
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<service name="cargo_vendor">
+   <summary>OBS Source Service to vendor all crates.io and dependencies for a Rust project</summary>
+   <description><![CDATA[This service extracts a Rust application source,
+  searches for a Rust application containing a Cargo.toml file,
+  download all crates.io and dependecies,
+  and creates a vendor.tar[.<tar compression>] to be committed allowing fully offline
+  builds of Rust applications.]]></description>
+   <parameter name="strategy">
+      <description>Legacy argument, no longer used. Values: cargo_vendor. Default: cargo_vendor</description>
+   </parameter>
+   <parameter name="method">
+      <description>Whether to use vendor or the registry. Default: vendor</description>
+      <allowedvalue>registry</allowedvalue>
+      <allowedvalue>vendor</allowedvalue>
+   </parameter>
+   <parameter name="src">
+      <description>Where to find sources. Source is either a directory or a source tarball AND cannot be both. Aliases: srctar, srcdir</description>
+   </parameter>
+   <parameter name="outdir">
+      <description>Where to output vendor.tar* and cargo_config if method is vendor and registry.tar* if method is registry.</description>
+   </parameter>
+   <parameter name="custom-root>
+      <description>Whether you want to manually set the root of the
+      project. Useful with a combination with `--manifest-path` (aliased as `--cargotoml`) or
+      `--no-root-manifest`.
+      </description>
+   </parameter>
+   <parameter name="update">
+      <description>Update dependencies or not. Default: true</description>
+      <allowedvalue>false</allowedvalue>
+      <allowedvalue>true</allowedvalue>
+   </parameter>
+   <parameter name="no-root-manifest">
+      <description>Available only if `--method` is set to registry. If a
+      project has no root manifest, this flag is useful for those situations
+      to set the manifest path manually. Useful in combination with
+      `--manifest-path` (aliased as `--cargotoml`) flag. Default: false
+      </description>
+      <allowedvalue>false</allowedvalue>
+      <allowedvalue>true</allowedvalue>
+   </parameter>
+   <parameter name="tag">
+      <description>Tag some files for multi-vendor and multi-cargo_config projects</description>
+   </parameter>
+   <parameter name="compression">
+      <description>What compression algorithm to use. Set to `not` if you just want a normal tarball with no compression. Default: zst</description>
+      <allowedvalues>zst</allowedvalues>
+      <allowedvalues>gz</allowedvalues>
+      <allowedvalues>xz</allowedvalues>
+      <allowedvalues>bz2</allowedvalues>
+      <allowedvalues>not</allowedvalues>
+   </parameter>
+   <parameter name="cargotoml">
+      <description>Other cargo manifest files to sync with vendor or registry. Behaviour between methods changes. Consult the documentation.</description>
+   </parameter>
+   <parameter name="i-accept-the-risk">
```

Comment:
I'd make it "i accept the risk" and then in the description say "you need to replace the spaces with '-' for this parameter to work. That way people don't copy-paste it blindly.